website2/.vuepress/local-components/SpeakerModal.vue

```diff
@@ -140,9 +142,12 @@ export default {
   position: relative;
   box-sizing: border-box;
   z-index: -2;
+
+  animation-timing-function: linear;
+  animation: modal-expand 1.5s, modal-opacity 1s;
```

Comment:
Would be cool if the animation is more snappy. How about 0.4s?

A good animation is one that doesn’t catch too much attention ah.

Reply:
I have decreased to 0.4s. But not sure that look cool or not.  😂